fixtures/definition-syntax-match/generic.json

```diff
@@ -457,7 +457,6 @@
             "123456n-12345678"
         ],
         "invalid": [
-            "",
```

Comment:
This line shouldn't be removed. This fixture is for testing definition syntax and an empty string is invalid. I commented before that a test which testing `An+B` parsing should be altered to exclude this case. The test I referred is actually re-using this fixture for its own purposes.